dex/fiatrates/oracle.go

```diff
@@ -84,27 +118,30 @@ func (o *Oracle) assets() []string {
 // Run starts goroutines that refresh fiat rates every source.refreshInterval.
 // This should be called in a goroutine as it's blocking.
 func (o *Oracle) Run(ctx context.Context, log dex.Logger) {
+	o.ctx = ctx
+	o.log = log
```

Comment:
I'm glad you added the quick add but I don't like the saved ctx too much. Could `func (o *Oracle) Rate(symbol string) float64` just accept a ctx? Saving log is fine imo.

https://go.dev/blog/context-and-structs

Reply:
> but I don't like the saved ctx too much.

yh, me too. your suggestion is okay.